docs/src/modules/ROOT/pages/integration/config-solver-properties.adoc

```diff
@@ -0,0 +1,52 @@
+////
+Quarkus and Spring Boot support the same configuration properties.
+All the properties are in this file, which can then be included multiple times.
+The {property_prefix} attribute is used for Quarkus properties.
+////
+
+{property_prefix}timefold.solver.<solverName>.solver-config-xml::
+A classpath resource to read the solver configuration XML.
+Defaults to `solverConfig.xml`.
+If this property isn't specified, that file is optional.
```

Comment:
What do you mean "optional"? If the property is not specified, but the file `solverConfig.xml` is still found, will it be used? And if the file is not found/not used, where do the defaults come frome?

(I'm not saying we should _change_ existing behavior, but we should _describe_ it.)

Reply:
It means something must be present on the classpath if something is specified.
I've changed the description to be more consistent.